_config/dev.yml

```diff
@@ -0,0 +1,5 @@
+---
+Name: mfa-devconfig
+---
+SilverStripe\MFA\Service\EnforcementManager:
```

Comment:
Missing the dev environment config?

Reply:
Argh sorry, debugging artefact - I had removed this to figure out why the config didn't apply. Readded now.